components/TipOfTheDay.js

```diff
@@ -1,47 +1,76 @@
 import styled from "styled-components";
-
+import Image from "next/image";
+import Herbie from "@/assets/Herbie.png";
 import Link from "next/link";
 
 const StyledCard = styled.div`
   display: flex;
-  flex-direction: column;
+  justify-content: center;
   align-items: center;
+  background: #fcfbf4;
+  flex-direction: column;
   gap: 10px;
-  padding: 20px;
+  padding: 15px 25px 25px 25px;
   border: 1px solid #ccc;
   border-radius: 5px;
+  border-color: black;
```

Comment:
Here you first used the border shorthand to describe the property of the border color and then you set the border color a second time. Defining it one time as intended makes it simpler.